src/NzbDrone.Core.Test/MediaFiles/DiskScanServiceTests/ScanFixture.cs

```diff
@@ -346,12 +345,12 @@ public void should_scan_dotHack_folder()
         [Test]
         public void should_exclude_osx_metadata_files()
         {
-            GivenSeriesFolder();
+            GivenArtistFolder();
 
             GivenFiles(new List<string>
                        {
                            Path.Combine(_artist.Path, "._24 The Status Quo Combustion.mp4").AsOsAgnostic(),
```

Comment:
Do mp4 files work as audio too or should this also be changed/removed?

Reply:
Should be changed